src/assets/styles/theme.js

```diff
@@ -0,0 +1,16 @@
+export const theme = {
+  colors: {
+    white: '#FFFFFF',
+    black: '#111111',
+    lightGrey: '#FBF9FC',
+    darkGrey: '#606060',
+    darkPurple: '#773795',
```

Comment:
Prawdę mówiąc w wielu projektach widziałam theme utworzone na zasadzie "nazwaKoloru: #kolor", więc w pracy pewnie zastaniesz własnie takie struktury. Natomiast warto zawsze spojrzeć na to trochę szerzej. Zamiast dawać zmiennym nazwy kolorów, można przypisać im bardziej "funkcjonalne" nazwy. I tak przykładowo:
textSecondary: '#FBF9FC', // use for regular text
textPrimary: '#606060', // use for headers
primary: '#773795', // primary UI elements
secondary: '#ECEFF7', // accent colors

W ten sposób theme faktycznie będzie mógł być "skórką" w której wystarczy podmienić wartosci by stworzyć np. dark theme lub gdy (co bardziej spotykane) klient stwierdzi że zamiast fioletowego robi rebranding wszystkiego na zielony :)